core/arch/arm/include/kernel/thread_private_arch.h

```diff
@@ -240,15 +183,11 @@ uint32_t thread_get_usr_sp(void);
 #endif /*ARM32*/
 
 /* Checks stack canaries */
```

Comment:
commit "core: split core/include/kernel/thread_private.h":
could you also remove the inline comment?

Reply:
Thanks, I'll fix.